.github/workflows/Security Vulnerability Check on Release Tags.yml

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Security Vulnerability Check on Release Tags
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  
+  pull_request:
+    branches: [main] 
+   
+env:
+  EDFI_ODS_IMP_TOKEN: ${{ secrets.REPO_DISPATCH_TOKEN }}
+  REPOSITORY_OWNER: ${{ GITHUB.REPOSITORY_OWNER }}   
+jobs:
+  Security-Vulnerability-Check-Each-Release-Tag:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        tag: ["v5.3-patch5","v6.1-patch3","v7.1-patch1"]   
```

Comment:
Can we temporarily add the tag `v5.3` just to see the pipeline fail? (It should detect a vulnerability in the _System.Data.SqlClient_ package).

Reply:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/actions/runs/7984100004/job/21800366216?pr=951#step:3:19  Yes It's failing and able to detect it

Reply:
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/assets/34613894/fdca19fb-e9f4-41cc-9f2e-a290ba56476e)